remote/remote.go

```diff
@@ -207,11 +207,24 @@ func newV1Image(keychain authn.Keychain, repoName string, platform imgutil.Platf
 			}
 			return nil, fmt.Errorf("connect to repo store '%s': %s", repoName, err.Error())
 		}
+		break
 	}
-
 	return image, nil
 }
 
+func isRetryable(err error) bool {
+	if err == io.EOF {
+		return true
+	}
+	if transportErr, ok := err.(*transport.Error); ok {
```

Comment:
I see that in the if condition on line 199, we also check that len(transportErr.Errors) > 0.
Do we really need it? And if we do, do we need to add it to the condition on line 219?